Source/santad/EventProviders/SNTEndpointSecurityManagerTest.mm

```diff
@@ -200,4 +200,128 @@ - (void)testSkipOtherESEvents {
   XCTAssertEqual(got.result, ES_AUTH_RESULT_ALLOW);
 }
 
+- (void)testRenameOverwriteRulesDB {
+  for (const NSString *testPath in @[ kEventsDBPath, kRulesDBPath ]) {
+    MockEndpointSecurity *mockES = [MockEndpointSecurity mockEndpointSecurity];
+    [mockES reset];
+    SNTEndpointSecurityManager *snt = [[SNTEndpointSecurityManager alloc] init];
+
+    XCTestExpectation *expectation = [self expectationWithDescription:@"Wait for response from ES"];
+    __block ESResponse *got = nil;
```

Comment:
No need for the nil initialization, objc variables are nil initialized by default.

__block ESResponse *got;